### ISSUE TITLE
High cpu usage after unlock

### ISSUE THREAD

**Issue by CactiChameleon9** [vicr123/tsscreenlock#1]
This has happened twice to me so far - after laptop lid is closed, and then reopening (and then logging back in) and then I notice after 10 minutes that my laptop fans have ramped up, and it is at the top of the list with htop.

I am running the shell on arch linux, if that helps.

**Comment by vicr123** [comment]
Hmm, that's interesting. Is this after your computer returns from suspend mode? It should theoretically quit once you've entered your password 🤔

**Comment by CactiChameleon9** [comment]
Yes, this is after my laptops suspends (I think). Thanks for the reply.

**Comment by CactiChameleon9** [comment]
I just did a small test... after doing meta+l and then logging back in the same behaviour occurs, meaning it may be nothing to do with the suspending.

**Comment by vicr123** [comment]
Do you hear any audio when you log back in?

**Comment by CactiChameleon9** [comment]
No, but I never do... and the shell crashes if I turn up my volume from my keyboard. Probably because I do not have pulseaudio installed (doing so now)

**Comment by CactiChameleon9** [comment]
OK, now that I have pulse audio, the issue seems to have stopped, so a try and catch may be necessary to stop it from erroring out if it a sound cannot be played... Hope that this helped.

**Comment by vicr123** [comment]
Thanks for the heads up, I'll look into it from here. :)
